department-of-veterans-affairs/va.gov-cms
# VACMS-6230 Add policies pathauto pattern back.

## Description

Closes #6230

## Testing done


## Screenshots


## QA steps
 - Go to /admin/content/bulk?title=&type=vamc_system_policies_page&moderation_state=All&taxonomy_entity_index_tid_depth=All
 - find the most recently edited policies pages at the top of the list
 - select them and perform the bulk operation to fix their alias.
 - [ ] validate that pages has a path that matches the system/policies pattern (Click on the pages to see them as the view is cached and will not reveal the new alias)


### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
